02 clase/Lanister.java

```diff
@@ -0,0 +1,8 @@
+public class Lannister {
+	private String motto = "A Lanister always pays his debts";
+	private int militaryForce = 20000;
```

Comment:
What do you thing about setting the value directly in the declaration? Also Lanister must extends of what class?